runtime/src/chplsys.c

```diff
@@ -52,6 +52,9 @@
 #include <errno.h>
 #include <unistd.h>
 
+#include <signal.h>
```

Comment:
I'm not sure keeping them in order is a worthwhile trade-off for adding more #ifdef __linux__. I think I would rather see all platform-specific headers included first and then all others. We could still have a group for platform-specific things...